terra-jupyter-r/Dockerfile

```diff
@@ -121,7 +121,7 @@ RUN pip3 -V \
 
 RUN R -e 'install.packages("BiocManager")' \
     ## check version
-    && R -e 'BiocManager::install(version="3.14", ask=FALSE)' \
+    && R -e 'BiocManager::install(version="3.15", ask=FALSE)' \
```

Comment:
you'll want to remove the version pinning in https://github.com/DataBiosphere/terra-docker/pull/319/files#diff-d6244546dc402de6ec2a9151ee2162627649b44b232f2375cae4dde47404773cL41 so that we get R 4.2 and not 4.1

Reply:
I don't think the `BiocManager::install(version = '3.15')` line has any bearing on the version of R being installed. 

R is being installed on lines 41 and 42. The version pinning needs to be removed there.